ads/google/a4a/traffic-experiments.js

```diff
@@ -107,8 +135,7 @@ function handleUrlParameters(win, experimentId, branches) {
         forceExperimentBranch(win, experimentId, branches.experiment);
         break;
       default:
-        dev().warn('a4a-config', 'Unknown a4a URL parameter: ',
-                 a4aParam[1]);
+        dev.warn('a4a-config', 'Unknown a4a URL parameter: ', arg);
```

Comment:
probably better to print the whole `a4aParam` for failures, in case `arg` is `undefined`